test/e2e/e2e_test.go

```diff
@@ -40,24 +40,6 @@ const (
 	// running and ready before any e2e tests run. It includes pulling all of
 	// the pods (as of 5/18/15 this is 8 pods).
 	podStartupTimeout = 10 * time.Minute
-
-	// minStartupPods is the minimum number of pods that will allow
-	// waitForPodsRunningReady(...) to succeed (i.e. WLOG if you know that
-	// "DNS", and "Prometheus" pods need to be running, you might set it to "2").
-	// More verbosely, that function
-	// checks that all pods in the cluster are both in a phase of "running" and
-	// have a condition of "ready": "true". It aims to ensure that the cluster's
-	// pods are fully healthy before beginning e2e tests. However, if there were
-	// only 0 pods, it would technically pass if there wasn't a required minimum
-	// number of pods. We expect every cluster to come up with some number of
-	// pods (which in practice is more than this number), so we have this
-	// minimum here as a sanity check to make sure that there are actually pods
-	// on the cluster (i.e. preventing a possible race with kube-addons). This
-	// does *not* mean that the function will succeed as soon as minStartupPods
-	// are found to be running and ready; it ensures that *all* pods it finds
-	// are running and ready. This is the minimum number it must find.
-	// TODO : Add command line option for this so that the number is non trivial.
-	minStartupPods = 0
```

Comment:
yeah... unfortunately the comment already didn't match the line following it.

maybe everything after "More verbosely" should be moved to the comment for waitForPodsRunningReady.